lib/concurrent.rb

```diff
@@ -11,6 +11,7 @@
 require 'concurrent/executors'
 require 'concurrent/utilities'
 
+require 'concurrent/atomic/atomic_markable_reference'
```

Comment:
We'll also need to update `lib/concurrent/file_map.rb` to make sure this class is packaged in the correct gem.